src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -224,46 +221,45 @@ function Test-SetAzureStorageAccount
         $stotype = 'Standard_GRS';
         $loc = Get-ProviderLocation ResourceManagement;
         $kind = 'Storage'
-        $encryptionServiceBF = "File,Blob"
 
         New-AzureRmResourceGroup -Name $rgname -Location $loc;
-        New-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype -Kind $kind -EnableEncryptionService $encryptionServiceBF -EnableHttpsTrafficOnly $true;
+        New-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype -Kind $kind -EnableHttpsTrafficOnly $true;
 
         Retry-IfException { $global:sto = Get-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname; }
         $stotype = 'StandardGRS';
-        Assert-AreEqual $sto.StorageAccountName $stoname;
-        Assert-AreEqual $sto.Sku.Name $stotype;
-        Assert-AreEqual $sto.Location $loc;
-        Assert-AreEqual $sto.Kind $kind;
-        Assert-AreEqual $sto.EnableHttpsTrafficOnly $true;
-        Assert-AreEqual $sto.Encryption.Services.Blob.Enabled $true
-        Assert-AreEqual $sto.Encryption.Services.File.Enabled $true
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual $true $sto.EnableHttpsTrafficOnly;
+        Assert-AreEqual $true $sto.Encryption.Services.Blob.Enabled
+        Assert-AreEqual $true $sto.Encryption.Services.File.Enabled
         
         $stos = Get-AzureRmStorageAccount -ResourceGroupName $rgname;
-        Assert-AreEqual $stos[0].StorageAccountName $stoname;
-        Assert-AreEqual $stos[0].Sku.Name $stotype;
-        Assert-AreEqual $stos[0].Location $loc;
-        Assert-AreEqual $sto.Kind $kind;
-        Assert-AreEqual $sto.EnableHttpsTrafficOnly $true;
-        Assert-AreEqual $sto.Encryption.Services.Blob.Enabled $true
-        Assert-AreEqual $sto.Encryption.Services.File.Enabled $true
+        Assert-AreEqual $stoname $stos[0].StorageAccountName;
+        Assert-AreEqual $stotype $stos[0].Sku.Name;
+        Assert-AreEqual $loc $stos[0].Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual $true $sto.EnableHttpsTrafficOnly;
+        Assert-AreEqual $true $sto.Encryption.Services.Blob.Enabled
+        Assert-AreEqual $true $sto.Encryption.Services.File.Enabled
 
         $stotype = 'Standard_LRS';
         # TODO: Still need to do retry for Set-, even after Get- returns it.
-        Retry-IfException { Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -StorageEncryption -DisableEncryptionService $encryptionServiceBF -EnableHttpsTrafficOnly $false }
+        Retry-IfException { Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -StorageEncryption -EnableHttpsTrafficOnly $false }
```

Comment:
Is there a test for `Set-AzureRmStorageAccount` that doesn't use `-StorageEncryption`?

Reply:
yes, the 'Set-AzureRmStorageAccount' in test case 'Test-NetworkRule' and 'Test-SetAzureStorageAccountStorageV2' don't have '-StorageEncryption'. 
Anyway, I have remove it from all test cases except 'Test-SetAzureRmStorageAccountKeySource', since it's not necessary in other cases.